src/components/SqlModal.svelte

```diff
@@ -74,23 +183,51 @@
       View SQL Query
     </button>
   </div>
-  <div slot="title">SQL Query</div>
+  <div slot="title">Explore the data...</div>
   <div>
     <p>
       The following SQL query can be copy/pasted and used in the BigQuery
       console to explore this data further:
     </p>
-    <textarea
-      bind:this={textarea}
-      spellcheck="false"
-      use:setHeight
-      readonly
-      wrap="off">
-      {getSql()}
-    </textarea>
-    <button on:click={copyTextarea}>Copy to clipboard</button>
-    {#if status}
-      <span contenteditable="true" bind:textContent={status} transition:fade />
+    {#if activeTab === 2}
+      <p>
+        Read <a
+          href="https://docs.telemetry.mozilla.org/cookbooks/main_ping_exponential_histograms.html">
+          Visualizing Percentiles of a Main Ping Exponential Histogram</a>
+        for more on the use of this query.
+      </p>
```

Comment:
👍

Reply:
Looking at the screenshot, I'm wondering if it would look better if we pasted this text into the SQL statement as a comment. It looks a little weird to have this just below the `... :`